apps/hermes/server/src/api/types.rs

```diff
@@ -261,7 +262,7 @@ pub struct ParsedPriceFeedTwap {
     /// This is a float value stored as a string to avoid precision loss.
     #[serde(with = "pyth_sdk::utils::as_string")]
```

Comment:
you probably don't need it anymore.

Reply:
Removed the serde as_string and [added utoipa's `decimal` feature instead which adds support for `rust_decimal.Decimal`.](https://github.com/juhaku/utoipa?tab=readme-ov-file#:~:text=decimal%3A%20Add%20support%20for%20rust_decimal%20Decimal%20type.%20By%20default%20it%20is%20interpreted%20as%20String.%20If%20you%20wish%20to%20change%20the%20format%20you%20need%20to%20override%20the%20type.%20See%20the%20value_type%20in%20component%20derive%20docs.). Ultimately it just treats it as a string anyway, so no functional difference :)